configuration/add-custom-certificates.adoc

```diff
@@ -0,0 +1,48 @@
+[id="add-custom-cert"]
+= Adding custom certificates
+include::modules/common-attributes.adoc[]
+:context: acs-config
+
+toc::[]
+
+[role="_abstract"]
+Learn how to use a custom TLS certificate with {product-title}. After you set up a certificate, users and API clients do not have to bypass the certificate security warnings when connecting to Central.
+
+
+[id="add-custom-security-cert"]
+== Adding a custom security certificate
+You can apply a security certificate during the installation or on an existing {product-title} deployment.
+
+//Prerequisites
+include::modules/custom-cert-prerequisites.adoc[leveloffset=+2]
+
+//During a new installation
+include::modules/custom-cert-new-install.adoc[leveloffset=+2]
+
+//On an existing installation
+include::modules/custom-cert-existing.adoc[leveloffset=+2]
+
+[id="configure-sensor-to-trust-cert"]
+== Configuring Sensor to trust custom certificates
+If you are using a custom certificate that isn't trusted globally, you must configure the Sensor to trust your custom certificate.
```

Comment:
isn't -> is not

Reply:
🤦🏼 I searched and fixed the ones with `'` but missed the `’` ones.